entgql/internal/todoplugin/gqlgen.yml

```diff
@@ -21,7 +21,11 @@ models:
   Duration:
     model:
       - entgo.io/contrib/entgql/internal/todo/ent/schema/durationgql.Duration
-  # Custom mapping from GraphQL `CategoryConfigInput` to `schematype.CategoryConfig`.
+  # Custom mapping from GraphQL `CategoryConfig` and
+  # `CategoryConfigInput` to `schematype.CategoryConfig`.
+  CategoryConfig:
+    model:
+      - entgo.io/contrib/entgql/internal/todo/ent/schema/schematype.CategoryConfig
```

Comment:
I revert the support for custom Go types because the GQL type was not defined by Ent. The user should manual mapping for it the gqlgen's config or use `@goModel` to GQL type.

Reply:
Can’t we auto inject the goModel directive?

Reply:
It can be done by using the plugin only. But when we write the output to the file, we should not modify user-defined files